src/services/jsDoc.ts

```diff
@@ -89,17 +89,14 @@ namespace ts.JsDoc {
         // Eg. const a: Array<string> | Array<number>; a.length

```

Comment:
Comment on 86 claims it's already deduping? Is the `forEachUnique` meant to do this? Why didn't that work

Reply:
So I thought the same thing (i.e. "whoever wrote this must be mistaken!) but reading that comment, I understand `forEachUnique` must dedupe across declarations, and something like, say, `(Array<string> | Array<number>)["length"]` has the same declarations between `Array<string>` and `Array<number>` for either of them.

Reply:
I could *technically* let the "verbatim string" checks handle this instead.